samples/cellular/nrf_cloud_rest_cell_location/README.rst

```diff
@@ -77,6 +79,22 @@ Check and configure the following Kconfig options for the sample:
 CONFIG_REST_CELL_LOCATION_DO_JITP - Enable prompt to perform JITP via REST
    This configuration option defines whether the application prompts the user for just-in-time provisioning on startup.
 
+.. _CONFIG_REST_CELL_CHANGE_CONFIG:
+
+CONFIG_REST_CELL_CHANGE_CONFIG - Enable changing request configuration
+   Set this to use the next combination of ``hi_conf`` and ``fallback`` flags after performing both single and multi-cell location requests.
+
+.. _CONFIG_REST_CELL_DEFAULT_HICONF_VAL:
+
+CONFIG_REST_CELL_DEFAULT_HICONF_VAL - Enable high confidence result
+   Enable a 95% confidence interval for the location, instead of the default 68%.
+
+.. _CONFIG_REST_CELL_DEFAULT_FALLBACK_VAL:
+
+CONFIG_REST_CELL_DEFAULT_FALLBACK_VAL - Enable fallback to coarse location
+   If enabled and the location of the cell tower or Wi-Fi access points cannot be found, return area-level location based on the cellular tracking area code.
+   Otherwise a 404 error will be returned indicating location is not known.
```

Comment:
Not sure I would mention 404 as that's not visible to the user of the sample, right?

```suggestion
   Otherwise an error will be returned indicating location is not known.
```